go.mod

```diff
@@ -27,11 +28,12 @@ require (
 	github.com/googleapis/gnostic v0.0.0-20170729233727-0c5108395e2d // indirect
 	github.com/gregjones/httpcache v0.0.0-20170728041850-787624de3eb7 // indirect
 	github.com/hashicorp/golang-lru v0.0.0-20160207214719-a0d98a5f2880 // indirect
+	github.com/hpcloud/tail v1.0.0 // indirect
 	github.com/imdario/mergo v0.0.0-20180608140156-9316a62528ac // indirect
 	github.com/json-iterator/go v0.0.0-20180701071628-ab8a2e0c74be // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
-	github.com/nxadm/tail v1.4.8 // indirect
+	github.com/onsi/ginkgo v1.8.0 // indirect
```

Comment:
This breaks e2e (`go test ./test/e2e` fails because `ginkgo.seed` is redefined), but we probably don’t care.

Reply:
yeah e2e pulls in the original `github.com/submariner-io/submariner/test/e2e/framework` which pulls in ginkgo v1. It doesn't matter anyway b/c there are no actual test cases, ie they're all pending.

Reply:
OK I removed the references to the **submariner** project in the e2e code and now at least `go test` runs.